metaci/build/autoscaling.py

```diff
@@ -95,9 +95,10 @@ def scale(self):
 
 class HerokuAutoscaler(Autoscaler):
     """Scale using Heroku worker dynos."""
+    API_ROOT = "https://api.heroku.com/apps"
 
     def scale(self):
-        url = f"https://api.heroku.com/apps/{settings.HEROKU_APP_NAME}/formation/worker"
+        url = f"{API_ROOT}/{settings.HEROKU_APP_NAME}/formation/{settings.WORKER_DYNO_NAME}"
```

Comment:
I think this will result in `NameError: name 'API_ROOT' is not defined`; class members aren't included in method scope in Python.

Reply:
heh yea you right